packages/util-endpoints/src/utils/getEndpointUrl.ts

```diff
@@ -5,7 +5,12 @@ import { evaluateExpression } from "./evaluateExpression";
 export const getEndpointUrl = (endpointUrl: Expression, options: EvaluateOptions): URL => {
   const expression = evaluateExpression(endpointUrl, "Endpoint URL", options);
   if (typeof expression === "string") {
-    return new URL(expression);
+    try {
+      return new URL(expression);
+    } catch (error) {
+      console.error(`Failed to construct URL with ${expression}`, error);
+      throw error;
+    }
```

Comment:
I believe this diff is also added for debugging, and can be removed?

Reply:
I want to provide more detail here even for production use, because otherwise the error is too generic "Invalid URL".